src/components/ChatForm.js

```diff
@@ -38,63 +39,64 @@ const SendChatButton = styled.button`
   height: 100%;
 
   &:hover {
-    background: #185FBD;
+    background: #185fbd;
   }
 
   &.disabled {
-    background: #DFDFDF;
-    color: #BABABA;
+    background: #dfdfdf;
+    color: #bababa;
   }
 `;
 
-export default function ChatForm({ handleChatSend, activeUser }) {
+export default function ChatForm({ currentUser, selectedRoom }) {
   const [message, setMessage] = useState('');
-  
+  const { sendMessage } = useRooms();
+
   /**
    * Event Handler: ChatInput/onChange - set state 'message' when input value changes
    * @param {object} e - InputEvent object triggered by input field value chagne
    */
   const handleChange = (e) => {
     setMessage(e.target.value);
-  }
+  };
 
   /**
    * Event Handler: ChatInput/onKeyPress - trigger send when user presses enter key
    * @param {object} e - KeyboardEvent object triggerd by keypress on ChatInput
    */
   const handleKeyPress = (e) => {
-    if(e.code === "Enter" && !e.isComposing) {
+    if (e.code === 'Enter' && !e.isComposing) {
       handleSend();
     }
-  }
+  };
 
   /**
    * Send message containing contens of the input value. Alert when input is emtpy
    */
   const handleSend = () => {
```

Comment:
```suggestion
  const handleSendChatButtonClick = () => {
```

네이밍과 실제 하는 역할이 다르죠?
지금 SendChatButton의 클릭 이벤트를 다루는 함수인데, 'click send chat button'을 그냥 'send'라는 동사로 추상화하면 함수의 네이밍이 함수가 하는 일을 정확하고 구체적으로 반영하지 못하는 것 같습니다.

`handleSendChatButtonClick`이 좋아 보이네요 ㅎㅎ